base/show.jl

```diff
@@ -322,6 +327,11 @@ function isidentifier(s::AbstractString)
     end
     return true
 end
+"""
+    isidentifier(s) -> Bool
+
+Returns whether symbol is vaild identifier.
```

Comment:
Could you replace "symbol" by "string"? `isidentifier` acts on strings, not symbols.

```
Returns whether the string `s` is a valid identifier in Julia code.

Note that Julia internally creates and uses symbols with names that are not valid identifiers, such as e.g. "asdf#123". This prevents naming conflicts with symbols corresponding to identifiers found in Julia source code.
```